From 615762ad1f0d7c226d658b94dcce39075a78dbc5 Mon Sep 17 00:00:00 2001
From: Vasyl Gello <vasek.gello@gmail.com>
Date: Wed, 5 Feb 2020 11:59:06 +0000
Subject: [PATCH] Bump Security String to 2020-02-05

Implemented:
============
CVE:            References:     Type:   Severity:       Updated AOSP versions:
CVE-2020-0005   A-141552859     EOP     High            8.0, 8.1, 9, 10
CVE-2020-0014   A-128674520     EoP     High            8.0, 8.1, 9, 10
CVE-2020-0015   A-139017101     EoP     High            8.0, 8.1, 9, 10
CVE-2020-0017   A-123232892     ID      High            8.0, 8.1, 9, 10
CVE-2020-0018   A-139945049     ID      High            8.0, 8.1, 9, 10
CVE-2020-0022   A-143894715     DoS     Moderate        10
                                RCE     Critical        8.0, 8.1, 9
CVE-2020-0026   A-140419401     EoP     High            8.0, 8.1, 9, 10

Not Implemented:
================
CVE:            References:     Type:   Severity:       Updated AOSP versions:
None

Not Applicable (platform source):
=================================
CVE:            References:     Type:   Severity:       Updated AOSP versions:
CVE-2019-2200   A-67319274      EoP     High            10
CVE-2020-0020   A-143118731     ID      High            10
CVE-2020-0021   A-141413692     DoS     High            10
CVE-2020-0023   A-145130871     ID      Critical        10
CVE-2020-0027   A-144040966     EoP     High            8.0, 8.1, 9, 10
CVE-2020-0028   A-122652057     ID      High            9

Change-Id: I819743b32c325877fb8bd284e3d4868b114c38c8
Signed-off-by: Vasyl Gello <vasek.gello@gmail.com>
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 8c04c8b995..4c8a88f9c5 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -131,7 +131,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2020-01-05
+    PLATFORM_SECURITY_PATCH := 2020-02-05
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.17.1

